shared/src/main/scala/com/eed3si9n/expecty/Expecty.scala

```diff
@@ -14,7 +14,7 @@
 
 package com.eed3si9n.expecty
 
-class Expecty extends Recorder[Boolean, Unit] {
+class Expecty(displayLoc: Boolean = false) extends Recorder[Boolean, Unit] {
```

Comment:
It may be more consistent to put this into val like `failEarly` and `showTypes`?

Reply:
🤦 hadn't notice those. Addressed : 73f89d0